tests/e2e/pageObjects/recommendations-page.ts

```diff
@@ -0,0 +1,33 @@
+import {Selector, t} from 'testcafe';
+
+export class RecommendationsPage {
+    veryUsefulVoteBtn = Selector('[data-testid=very-useful-vote-btn]').nth(0);
+    usefulVoteBtn = Selector('[data-testid=useful-vote-btn]').nth(0);
+    notUsefulVoteBtn = Selector('[data-testid=not-useful-vote-btn]').nth(0);
+    recommendationsFeedbackBtn = Selector('[data-testid=recommendation-feedback-btn]');
+
+    async voteForVeryUsefulAndVerifyDisabled(): Promise<void> {
```

Comment:
please add this function description into comment above, like here:
![image](https://user-images.githubusercontent.com/90257586/214080495-5700f0fb-b0e9-47d0-b6c0-54f8666b7fca.png)